README.md

```diff
@@ -17,29 +17,59 @@ limitations under the License.
 
 [<img src="https://geode.apache.org/img/apache_geode_logo.png" align="center"/>](http://geode.apache.org)
 
-[![Build Status](https://travis-ci.org/apache/geode-examples.svg?branch=develop)](https://travis-ci.org/apache/geode-examples) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) 
+[![Build Status](https://travis-ci.org/apache/geode-examples.svg?branch=develop)](https://travis-ci.org/apache/geode-examples) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 # Apache Geode examples
 
-This is the home of Apache Geode examples that are bundled with the project. Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us about your suggestions at [dev@geode.apache.org](mailto:dev@geode.apache.org) or submit a [pull request](https://github.com/apache/geode/pull/new/develop).
+This is the home of Apache Geode examples that are bundled with the project.
+Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us
+about your suggestions at [dev@geode.apache.org](mailto:dev@geode.apache.org)
+or submit a [pull request](https://github.com/apache/geode/pull/new/develop).
 
-## Example requirements
+## Adding a new example
 
-All examples:
+Follow this approach to add a new example:
 
-*  Need to be testable. Use unit tests, integration tests or whatever is applicable. Tests will run through the project's CI.
```

Comment:
Are we removing the requirement for the example to be testable?  I'd push for examples to be contributed with tests or only merged and redistributed when they have tests.